lib/srfi/26/expand-cut.scm

```diff
@@ -0,0 +1,46 @@
+(define (expand-cut id close? list<argument-or-slot>)
+
+  (define (gs s) (gensym (string->uninterned-symbol s)))
```

Comment:
Using `gensym` directly, as in `(gensym 'slot)`, is the preferred style.  It isn't clear why the call to `string->uninterned-symbol` is there.

Reply:
Original implementation comes from my library of Gambit utils, which just uses define-runtime-macro everywhere instead of define-macro and stores all syntax as a part of the runtime.  My original thought is this might save permanently allocating a symbol or two, although I'm not sure if that's accurate (for this version it isn't helpful and will change).